bin/alluxio

```diff
@@ -181,7 +181,7 @@ function main {
   if [[ $- == *x* ]]; then
     LAUNCHER="bash -x"
   fi
-  BIN=$(cd "$( dirname "$( readlink "$0" )" )"; pwd)
+  BIN=$(cd "$( dirname "$( readlink "$0" || echo "$0" )" )"; pwd)
```

Comment:
Shall we define this a function as it becomes complicated and used multiple times.